scipy/sparse/tests/test_csc.py

```diff
@@ -96,3 +96,30 @@ def test_argmax_overflow(ax):
         ii, jj = np.max(idx), np.argmax(idx)
 
     assert A[ii, jj] == A[-2, -2]
+
+def test_broadcast_to():
```

Comment:
This test code is mostly duplicate between `test_csc.py` and `test_csr.py`. I think the two can be merged fairly easily, and moved to `test_base.py`. Writing the iteration like:

```python
for container in (csr_matrix, csr_array, csc_matrix, csc_array):
```

Probably worth doing that before merging.